server.py

```diff
@@ -158,19 +158,22 @@ def delete_recommendations(id):
         conn.execute("DELETE FROM recommendations WHERE id=%d" % id)
     return '', HTTP_204_NO_CONTENT
 
+######################################################################
+# ACTION - UPDATE PRIORITY WHEN RECOMMENDATION IS CLICKED
+######################################################################
 @app.route('/recommendations/<int:id>/clicked', methods=['PUT'])
 def increase_priority(id):
+    if get_recommendations(id).status_code == 404:
+        message = {'error': 'Recommendation with id: %s was not found' % str(id)}
+        return reply(message, HTTP_404_NOT_FOUND)
```

Comment:
166 -168 should go below the in-function comments (169-171) probably should delete those for consistency..